CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 ## Unreleased
+## 1.159.0 (January, 15 2024)
```

Comment:
Update the version and date

Reply:
updated version to latest 1.161.0, and current date February, 15 2024